src/components/pages/landing/header/index.tsx

```diff
@@ -32,11 +39,11 @@ const Header: React.FC<{topic?: string; customer?: any}> = ({
     </>
   )
 
-  if (viewer || customer) {
+  if (viewer || (customer && !isMember(viewer, customer))) {
     Offer = Join
   }
 
-  if (holidaySaleOn && (viewer || customer)) {
```

Comment:
@vojtaholik I think handling the holiday sale flag should be its own logic because this gets kind of confusing as it is